src/_playground/App.scss

```diff
@@ -73,10 +73,29 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
             overflow-y: scroll;
             background-color: white;
             padding: 2rem;
+          
+            .frDocs-Content__githubLogo {
+                float: right;
+                z-index: 100;
+                cursor: pointer;
+                width: 2rem;
+                height: 2rem;
+                font-size: 1rem;
+
+                svg {
+                    width: 2rem;
+                    height: 2rem;
+                }
+            }
 
             .frDocs-Content__header, .frDocs-markdown > h1 {
                 margin: 0 0 12px;
                 font-size: 2.2rem;
+                line-height: 2.2rem;
```

Comment:
TIP: If you are desiring the line height to match the font size, you could just enter it as `line-height: 1;` which is a ratio setting so if the font size ever changes, the line height will change as well.  Makes for easier maintenance going forward.